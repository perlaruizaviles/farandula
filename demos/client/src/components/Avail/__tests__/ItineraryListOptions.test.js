import React from 'react';
import {render} from 'enzyme';
import ItineraryListOptions from '../ItineraryListOptions';

describe('Rendering AirlegDetail ', () => {

  const tree = render(<ItineraryListOptions />);
  
  it('Should create an snapshot for AirlegDetail', () => {
    expect(tree).toMatchSnapshot();
  });
});