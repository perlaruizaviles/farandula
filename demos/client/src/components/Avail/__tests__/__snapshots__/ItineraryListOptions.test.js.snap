exports[`Rendering AirlegDetail  Should create an snapshot for AirlegDetail 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "aria-expanded": "false",
          "class": "ui button floating labeled icon dropdown",
          "role": "listbox",
          "tabindex": "0",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "text",
            },
            "children": Array [],
            "name": "div",
            "next": Object {
              "attribs": Object {
                "aria-hidden": "true",
                "class": "sort icon",
              },
              "children": Array [],
              "name": "i",
              "next": Object {
                "attribs": Object {
                  "class": "menu transition",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "item",
                      "role": "option",
                    },
                    "children": Array [
                      Object {
                        "data": " price-low-to-high ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "div",
                    "next": Object {
                      "attribs": Object {
                        "class": "item",
                        "role": "option",
                      },
                      "children": Array [
                        Object {
                          "data": " price-high-to-low ",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "div",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "tag",
                    },
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                  },
                  Object {
                    "attribs": Object {
                      "class": "item",
                      "role": "option",
                    },
                    "children": Array [
                      Object {
                        "data": " price-high-to-low ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "div",
                    "next": null,
                    "parent": [Circular],
                    "prev": Object {
                      "attribs": Object {
                        "class": "item",
                        "role": "option",
                      },
                      "children": Array [
                        Object {
                          "data": " price-low-to-high ",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "div",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                    },
                    "type": "tag",
                  },
                ],
                "name": "div",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
          },
          Object {
            "attribs": Object {
              "aria-hidden": "true",
              "class": "sort icon",
            },
            "children": Array [],
            "name": "i",
            "next": Object {
              "attribs": Object {
                "class": "menu transition",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "item",
                    "role": "option",
                  },
                  "children": Array [
                    Object {
                      "data": " price-low-to-high ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "div",
                  "next": Object {
                    "attribs": Object {
                      "class": "item",
                      "role": "option",
                    },
                    "children": Array [
                      Object {
                        "data": " price-high-to-low ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "div",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                },
                Object {
                  "attribs": Object {
                    "class": "item",
                    "role": "option",
                  },
                  "children": Array [
                    Object {
                      "data": " price-high-to-low ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "div",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "item",
                      "role": "option",
                    },
                    "children": Array [
                      Object {
                        "data": " price-low-to-high ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "div",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                  },
                  "type": "tag",
                },
              ],
              "name": "div",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "text",
              },
              "children": Array [],
              "name": "div",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
            },
            "type": "tag",
          },
          Object {
            "attribs": Object {
              "class": "menu transition",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "item",
                  "role": "option",
                },
                "children": Array [
                  Object {
                    "data": " price-low-to-high ",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "div",
                "next": Object {
                  "attribs": Object {
                    "class": "item",
                    "role": "option",
                  },
                  "children": Array [
                    Object {
                      "data": " price-high-to-low ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "div",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
              },
              Object {
                "attribs": Object {
                  "class": "item",
                  "role": "option",
                },
                "children": Array [
                  Object {
                    "data": " price-high-to-low ",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "div",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "item",
                    "role": "option",
                  },
                  "children": Array [
                    Object {
                      "data": " price-low-to-high ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "div",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                },
                "type": "tag",
              },
            ],
            "name": "div",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "aria-hidden": "true",
                "class": "sort icon",
              },
              "children": Array [],
              "name": "i",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "text",
                },
                "children": Array [],
                "name": "div",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
              },
              "type": "tag",
            },
            "type": "tag",
          },
        ],
        "name": "div",
        "next": null,
        "parent": null,
        "prev": null,
        "root": [Circular],
        "type": "tag",
      },
    ],
    "name": "root",
    "next": null,
    "parent": null,
    "prev": null,
    "type": "root",
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xmlMode": false,
  },
}
`;
